code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -1682,6 +1682,46 @@
 		body_parts = list(BP_HEAD)
 		species_allowed = list(SPECIES_TESHARI)
 
+	//Diona stuff.
+
+	diona_leaves

```

Comment:
This is how the rest of the file seems to work, what's wrong here and what should it be changed to exactly?

Reply:
See https://github.com/PolarisSS13/Polaris/wiki/Code-Standards#typepaths

Reply:
Nominally if the rest of the file is in relative pathing, that you correct all of it to absolute, but in the interest of chewing through as many prs as I can tonight I don't think it's strictly necessary. But anything that you do already change is required to be in absolute pathing

Reply:
After some fiddling around, I still can't figure out what the pathing for hair or markings *should* be that would work. Can you give me 1 example?

Reply:
I believe this one should be /datum/sprite_accessory/marking/diona_leaves

Reply:
he's asking for you to take the time as you adding those to also change them into absolute pathing instead of relative, if possible.

Reply:
The correct syntax is `/datum/sprite_accessory/marking/dion_leaves`, and then the indentation is reduced by one (i.e., no indentation on the typepath). Since absolute pathing is the standard syntax we use (and tbh I wasn't aware this file wasn't following it), the definition of basically any other object is a viable template for the formatting.